pages/api/delegates/legacy.ts

```diff
@@ -0,0 +1,204 @@
+/*
+
+SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+
+*/
+
+import { NextApiRequest, NextApiResponse } from 'next';
+import { fetchDelegates } from 'modules/delegates/api/fetchDelegates';
+import { DelegatesAPIResponse } from 'modules/delegates/types';
+import { DEFAULT_NETWORK, SupportedNetworks } from 'modules/web3/constants/networks';
+import withApiHandler from 'modules/app/api/withApiHandler';
+import validateQueryParam from 'modules/app/api/validateQueryParam';
+
+/**
```

Comment:
do we want to keep this legacy thing?

if we want to version the API i suggest we add a v1/v2 folder or route, 

routes mapping can be defined in next.config, 

but i think we should check with whoever team is using our API endpoints, ,and notify them of the migration

if the data structure changed , if not, is okay